ops/terraform/environments/scratch/scratch.tf

```diff
@@ -0,0 +1,15 @@
+
+// Forward-declare variables pulled in from the modules' variable declarations (in their .tf
+// files or inline in the module definition)
+variable "aou_env" {}
+variable "reporting_dataset_id" {}
+variable "project_id" {}
+variable "credentials_file" {}
+
+module "aou_rw_scratch_env" {
```

Comment:
TODO: make consistent module names. 

Also, I want one more level of abstraction for the whole depoyable AoU "stack", which will be the union of reporting, alerting, tiers, and whatever else. I think I might as well bit the bullet and put that in now, so I get a clear picture of how it will work. (Possibly with a trivial second inner module.)